apps/ecparam.c

```diff
@@ -229,9 +229,11 @@ int ecparam_main(int argc, char **argv)
         *p = OSSL_PARAM_construct_end();
 
         if (OPENSSL_strcasecmp(curve_name, "SM2") == 0)
-            gctx_params = EVP_PKEY_CTX_new_from_name(NULL, "sm2", NULL);
+            gctx_params = EVP_PKEY_CTX_new_from_name(app_get0_libctx(), "sm2",
+                    app_get0_propq());
```

Comment:
Normally the indentation on these multi line params lines up to the character after (

Reply:
I'll fix that in a second.